rust/core-lib/src/editor.rs

```diff
@@ -747,11 +747,7 @@ impl Editor {
         view.invalidate_styles(&self.text, start, end_offset);
     }
 
-    pub fn plugin_get_data(&self, start: usize,
-                           unit: TextUnit,
-                           max_size: usize,
-                           rev: RevToken) -> Option<GetDataResponse> {
-        let _t = trace_block("Editor::plugin_get_data", &["core"]);
+    pub (crate) fn get_rev(&self, rev: RevToken) -> Option<Cow<Rope>> {
```

Comment:
Rust style has solidified on having no space between `pub` and `(crate)`. I was also writing this like you are, but I'm changing to `pub(crate)` now.